utils/cliutils/commandsflags.go

```diff
@@ -953,6 +956,10 @@ var flagsMap = map[string]cli.Flag{
 		Name:  nugetV2,
 		Usage: "[Default: false] Set to true if you'd like to use the NuGet V2 protocol when restoring packages from Artifactory.` `",
 	},
+	vcsFallback: cli.BoolTFlag{
+		Name:  vcsFallback,
+		Usage: "[Default: true] Set to false if you don't like to download Go packages from the VCS in case they are missing in Artifactory.` `",
```

Comment:
I suggest we rename the flag to ```no-fallback``` with a ```false``` default value. This means that by default, we will fallback. I think this provides the best user experience. People who are concerned about the implications of this, can always add the flag.
So let's change the flag description to - 
```
[Default: false] Set to true if you'd like to download Go packages from the VCS, in case they are missing in Artifactory
``` 
Let's remember to also modify the code that reads the flag with the new default value, as well as the test.